espaloma/graphs/deploy.py

```diff
@@ -88,6 +88,17 @@ def openmm_system_from_graph(
         sys = ff.create_openmm_system(
             g.mol.to_topology(), charge_from_molecules=[g.mol]
         )
+        
+    elif charge_method == "am1-bcc":
+        g.mol.assign_partial_charges()
+        sys = ff.create_openmm_system(
+            g.mol.to_topology(), charge_from_molecules=[g.mol]
+        )
+        
+    elif charge_method == "keep":
+        sys = ff.create_openmm_system(
+            g.mol.to_topology(), charge_from_molecules=[g.mol]
+        )
```

Comment:
What does the `else:` behavior bellow do? Shouldn't we raise an exception for the `else:` case because the charge method is not recognized?

Reply:
for `else` it would use the default behavior of `ff.create_openmm_system` which is to use `am1-bcc` as well?

Reply:
I think we want to avoid `else:`, since we don't want to have typos in `charge_method` silently give weird behavior---typos in `charge_method` should raise a `ValueError`.